src/tests/system/tests/test_sss_ssh_knownhosts.py

```diff
@@ -0,0 +1,122 @@
+"""
```

Comment:
Should this have been includeded in this PR?  I do not see it in the other one here:
https://github.com/SSSD/sssd/pull/7160

Reply:
So I made the mistake of simply rebasing master on top of 2-9... undid a lot of changes, This PR should be easier to review now.